extension.json

```diff
@@ -161,6 +168,14 @@
 			"description": "Array. Memcached servers to use for clearing memcached keys when a wiki is deleted or renamed.",
 			"value": []
 		},
+		"MirahezeMagicRequestWikiExtensions": {
+			"description": "Array. Extensions to allow to select directly from RequestWiki.",
+			"value": [ "pageimages" ]
+		},
+		"MirahezeMagicRequestWikiSkins": {
+			"description": "Array. Skins to allow to select directly from RequestWiki.",
```

Comment:
Clarify message to something like "Skins to allow to be selected as the default skin directly from RequestWiki"

Reply:
That is a great idea! Thanks!

Reply:
Done